p2p/bootstrap.go

```diff
@@ -12,7 +12,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const dhtProtocolID = protocol.ID("/0x-mesh-dht/version/1")
+const dhtProtocolID = protocol.ID("/0x-mesh-dht/version/2")
```

Comment:
We need to keep the DHT protocol ID the same. Otherwise peer discovery won't work due to the fact that our bootstrap nodes are using `version/1`. The alternative is to spin up new bootstrap nodes, but I don't think there's a strong reason to do that.

Reply:
Good catch.